controllers/teams.js

```diff
@@ -1080,6 +1083,8 @@ router.get('/:teamId/members', async (req, res, next) => {
 			} else {
 				actions = addButtonTrash(actions);
 			}
+			const userRoles = roles.filter((r) => user.userId.roles.find((id) => id === r._id));
+			const canBeTeamAdmin = userRoles.some((role) => role.permissions.some((perm) => perm === 'CAN_BE_TEAM_ADMIN'));
```

Comment:
Codacy found an issue: [This line has a length of 123. Maximum allowed is 120.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-client/pullRequest?prid=6045526)